Packs/PrismaCloud/Layouts/layoutscontainer-Prisma_Cloud_-_Network_Incident_Layout.json

```diff
@@ -0,0 +1,895 @@
+                                "startCol": 0
+                            },
+                            {
+                                "endCol": 4,
+                                "fieldId": "cveid",
+                                "height": 53,
+                                "id": "4c9733c0-f033-11ee-ac36-c7fff22f5e19",
+                                "index": 0,
+                                "sectionItemType": "field",
+                                "startCol": 2
+                            },
+                            {
+                                "dropEffect": "move",
+                                "endCol": 4,
+                                "fieldId": "cvss",
+                                "height": 53,
+                                "id": "23e9f930-d481-11ee-9421-a33fe2480cb6",
+                                "index": 1,
+                                "listId": "zihqpezlwt-5cf18aa0-d480-11ee-9996-fd1cc96b85a8",
+                                "sectionItemType": "field",
+                                "startCol": 2
+                            },
+                            {
+                                "endCol": 4,
+                                "fieldId": "cvedescription",
+                                "height": 53,
+                                "id": "b2269b60-f018-11ee-91a8-4bbcbf7a699c",
+                                "index": 2,
+                                "sectionItemType": "field",
+                                "startCol": 2
+                            },
+                            {
+                                "dropEffect": "move",
+                                "endCol": 4,
+                                "fieldId": "cvepublished",
+                                "height": 53,
+                                "id": "a4a4a900-f018-11ee-91a8-4bbcbf7a699c",
+                                "index": 3,
+                                "listId": "zihqpezlwt-5cf18aa0-d480-11ee-9996-fd1cc96b85a8",
+                                "sectionItemType": "field",
+                                "startCol": 2
+                            },
+                            {
+                                "endCol": 4,
+                                "fieldId": "vulnerableproduct",
+                                "height": 106,
+                                "id": "a13747a0-f018-11ee-91a8-4bbcbf7a699c",
+                                "index": 4,
+                                "sectionItemType": "field",
+                                "startCol": 2
+                            }
+                        ],
+                        "maxW": 3,
+                        "minH": 1,
+                        "moved": false,
+                        "name": "Investigation Results",
+                        "static": false,
+                        "w": 2,
+                        "x": 1,
+                        "y": 0
+                    },
+                    {
```

Comment:
Please add the 'User Risk Level' alert field and map the ${Core.RiskyUser.risk_level} into it.